pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "skodaconnect"
+authors = [{name = "lendy007", email = "lendik@gmail.com"}]
```

Comment:
Might be time to change this 😁

Reply:
Yeah. Sounds like a good moment. Will update.